kitsune/questions/cron.py

```diff
@@ -151,3 +153,56 @@ def escalate_questions():
         escalate_question.delay(question.id)
 
     return len(qs_no_replies_yet)
+
+
+@cronjobs.register
+def report_employee_answers():
+    """Send an email about employee answered questions.
+
+    We report on the users in the "Support Forum Tracked" group.
+    We send the email to the users in the "Support Forum Metrics" group.
+    """
+    tracked_group = Group.objects.get(name='Support Forum Tracked')
+    report_group = Group.objects.get(name='Support Forum Metrics')
+
+    tracked_users = tracked_group.user_set.all()
+    report_recipients = report_group.user_set.all()
+
+    if len(tracked_users) == 0 or len(report_recipients) == 0:
+        return
+
+    yesterday = date.today() - timedelta(days=1)
+    day_before_yesterday = yesterday - timedelta(days=1)
+
+    # Total number of questions asked the day before yesterday
+    questions = Question.objects.filter(
+        creator__is_active=True,
+        created__gte=day_before_yesterday,
+        created__lt=yesterday)
+    num_questions = questions.count()
+
+    # Total number of answered questions day before yesterday
+    num_answered = questions.filter(num_answers__gt=0).count()
+
+    # Total number of questions answered by user in tracked_group
+    num_answered_by_tracked = Answer.objects.filter(
+        question__in=questions,
+        creator__in=tracked_users).values_list('question_id').distinct().count()
+
+    email_subject = 'Support Forum answered report for {date}'.format(date=day_before_yesterday)
+    email_body = """
+    Date: {date}
+    Number of questions asked: {num_questions}
+    Number of questions answered: {num_answered}
+    Number of questions with answer from 'Support Forum Tracked' users: {num_answered_by_tracked}
+    """.format(
```

Comment:
I just realize this is probably going to look really weird in a plain text email, because it will have lots of extra space. Or does `send_email` deal with that?